Zend/tests/fibers/get-return.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test fiber return value
+--FILE--
+<?php
+
+$fiber = new Fiber(function (): int {
+    $value = Fiber::suspend(1);
+    return $value;
+});
+
+$value = $fiber->start();
+var_dump($value);
+var_dump($fiber->resume($value + 1));
```

Comment:
It's good to use something like `$value . "x"` in such cases, to also test whether you are refcounting correctly. Gotta be careful as any literals are interned.

Reply:
Updated in 9ba9e3729ecf73ca170739b5c4e712ef3a953f53.